README.md

```diff
@@ -220,6 +221,36 @@ In the example above, the `Target` attribute is specified three different ways,
 
 One of the primary goals of the WebLogic Deploy Tooling is to support a sparse model where the user can specify just the configuration needed for a particular situation.  What this implies varies somewhat between the tools but, in general, this implies that the tools are using an additive model.  That is, the tools add to what is already there in the existing domain or domain templates (when creating a new domain) rather than making the domain conform exactly to the specified model.  Where it makes sense, a similar, additive approach is taken when setting the value of multi-valued attributes.  For example, if the model specified the cluster `mycluster` as the target for an artifact, the tooling will add `mycluster` to any existing list of targets for the artifact.  While the development team has tried to mark attributes that do not make sense to merge accordingly in our knowledge base, this behavior can be disabled on an attribute-by-attribute basis, by adding an additional annotation in the knowledge base data files.  The development team is already thinking about how to handle situations that require a non-additive, converge-to-the-model approach, and how that might be supported, but this still remains a wish list item.  Users with these requirements should raise an issue for this support.
 
+### Declaring Named MBeans to Delete
+
+With WebLogic Deploy Tooling release 1.3.0, you can specify named items in the model to be deleted using the Create Domain, Update Domain, Deploy Applications Tools.  Named items are those that have multiple instances that are distinguished by user-provided names, such as managed servers, datasources, and security realms.  Items to be deleted are prepended with an exclamation point (!) in the model. 
```

Comment:
Update Domain, Deploy Applications Tools. -> Update Domain, and Deploy Applications Tools.
datasources -> data sources ((adj) (n) Two words.)

Reply:
Changed these + one more data source